using-mobile-extensions/mobile-core/identity/README.md

```diff
@@ -81,7 +81,9 @@ cordova plugin add https://github.com/adobe/cordova-acpcore.git
 {% endtab %}
 
 {% tab title="Unity" %}
-### C\#
+
+### C#
+After importing the ACPCore.unitypackage . In the MainSceneScript the Identity extension can be added by using
```

Comment:
The first sentence `After importing the ACPCore.unitypackage .` seems incomplete. Should it be combined with the following sentence?

Reply:
ya